Update Commands (#89)
DynoCC/Dyno-Custom-Commands: Fun Commands/Mini-Games/Dyno-RPG/README.md

---

# Dyno RPG
**Dyno RPG** is a RPG Game made using Dyno Custom Commands and inspired by the infection minigame and a few other custom commands. It is planned to be a community project to be built into something everybody can enjoy. As such everyone is welcome to add their own commands to it as they desire.

> ## How to

Reviewer: This is great, although it seems to be more of a best practices section than a how to section.

Author: Yeah, it kinda is to an extent tbh. Dyno RPG is more or less designed to be user choice in the most part to make it as fun as possible. Some roles are strict but otherwise the game is more or less designed to have each roles limits chosen by the server staff.

A simple guide for setting up roles and channels for the commands to work as intended

- The `divine-council` channel should be only be useable by serverMods as it is the channel required for using the Divine Council commands.

- When creating the roles for the commands, make sure to determine which roles have which effects. The effects will be entirely up to the ones setting it up, so know that this RPG is best when each role has a specific effect on the one carrying it. 

- The `Adventurer` role is the base role for playing Dyno RPG.

- The channel `the-crypt` was created for use with the `Deceased` role. All people who carry this role are to be considered dead and are only able to continue chatting in here until they have been raised. This channel should be hidden by default to all except the `Deceased` and the `divine-council`.

> ## Avelon Commands

Reviewer: This should be "Dyno RPG Commands", since it seems this module is no longer called Avelon.

> A Full list of available commands for use with **Dyno RPG**

Reviewer: ```suggestion
A full list of available commands for use with **Dyno RPG**
```

**Registration Commands**

This command is used to join the world of Avelon

 - `register`: Grants the **Adenturer** role and enters you into the forsaken world of **Avelon**.

>  - `unregister`: Remove the Adventurer role and leave the forsaken world of **Avelon**.

Reviewer: ```suggestion
 - `unregister`: Removes the **Adventurer** role and frees you of the forsaken world of **Avelon**.
```
Just a general typo fix here.

**Attack Commands**

These commands can be used to attack other players, but be warned some come with *side effects*

 - `stab`: Stab another Adventurer 
 - `blight`: Cast Blight on another Adventurer
 - `sacrifice`: Sacrifice another Adventurer to a random deity.
 - `infect:` Infect another Advenurer.
 - `fester`: Cast Fester on another Adventurer. Can only be used by Infected Adventurers.
 - `plague`: Cast a plague infecting all Adventurers. Can only be used by infected Adventurers.
 - `deathmatch`: Challenge an Adventurer to a deathmatch.

>     - `cointoss`: One of the available deathmatches. Use this command when the coind toss deathmatch is chosen.

Reviewer: ```suggestion
    - `cointoss`: One of the available deathmatches. Use this command when the coin toss deathmatch is chosen.
```

These commands can only be used by the Divine Council *[aka serverMods]*

 - `divinecure`: Cures all Adventurers of all ailments.

>  - `dyno-rpg [enable|disable]`: Enable or Disable Dyno RPG Commands.

Reviewer: ```suggestion
 - `dyno-rpg [enable|disable]`: Enable or Disable **all** Dyno RPG Commands.
```
If this completely disables the module, than that likely should be marked very clearly.
